tests/refs/reflog/reflog_helpers.c

```diff
@@ -69,14 +125,7 @@ void reflog_print(git_repository *repo, const char *reflog_name)
 
 	for (idx = 0; idx < git_reflog_entrycount(reflog); idx++) {
 		const git_reflog_entry *entry = git_reflog_entry_byindex(reflog, idx);
-		git_buf buf = GIT_BUF_INIT;
-		char old_oid[GIT_OID_HEXSZ], new_oid[GIT_OID_HEXSZ];;
-		git_oid_tostr((char *)&old_oid, GIT_OID_HEXSZ, git_reflog_entry_id_old(entry));
-		git_oid_tostr((char *)&new_oid, GIT_OID_HEXSZ, git_reflog_entry_id_new(entry));
-
-		git_buf_printf(&buf, "%ld: %s %s %s %s\n", idx, old_oid, new_oid, "somesig", git_reflog_entry_message(entry));
-		fprintf(stderr, "%s", git_buf_cstr(&buf));
-		git_buf_free(&buf);
+		fprintf(stderr, "%s\n", reflog_entry_tostr(entry));
```

Comment:
This is a memory leak as `reflog_entry_tostr` returns a string detached from its buffer. I would instead propose an interface `int reflog_entry_tostr(git_buf *out, const git_reflog_entry *entry)`, which makes the need for memory management more obvious.